css_tasks/task9/scss/_tabs.scss

```diff
@@ -0,0 +1,126 @@
+@import "variables";
+@import "mixins";
+.tabs {
+  background: $mainBackgroundColor;
+}
+.tabs-content {
+  @include flex;
+  &__rectangles {
+    @include standardDesktopPadding;
+    // width: 10%;
+    flex-grow: 1;
+    .tab-rectangle {
+      @include button;
+      @include flex;
+      justify-content: center;
+      align-items: center;
+      width: 7rem;
+      height: 7rem;
+      margin-bottom: 2px;
+      background: $secondaryBackgroundColor;
+      text-transform: uppercase;
+      cursor: pointer;
+      &:first-child {
```

Comment:
it is not first child it is active tab which can change

Reply:
ok, will make active class and  add it via JS in the future